### DISCUSSION TITLE
flamesheet with a very small grid size

### DISCUSSION THREAD

**Discussion by Alex-jian522** [AMReX-Combustion/PeleLMeX#347] [General]
Hello,
When I use PeleLMeX to simulate flamesheet cases with a very small uniform grid size of 5e-5m (0.025 flamethickness, to resolve a very small disturbance). The code report like 
![c63362e20a126799a729433fc8e678a](https://github.com/AMReX-Combustion/PeleLMeX/assets/72745362/e2c54037-7b63-4c34-8519-0a4c09167272)
And when I use twice the size (0.05 flamethickness), the simulation can run and the results are correct.
What may be the reason for this problem? I use H2 as fuel and an 1D data for initilization.

**Comment by baperry2** [comment]
There is some information on troubleshooting this type of issue in the documentation, so try out these steps: https://amrex-combustion.github.io/PeleLMeX/manual/html/Troubleshooting.html#linear-solver-failure

**Reply by Alex-jian522** [reply]
Thanks for your answer. This document is very clear and solved my problem！
